lib/killbill/helpers/active_merchant/utils.rb

```diff
@@ -1,3 +1,6 @@
+require 'thread'
+require 'thread_safe'
```

Comment:
well that's a thing - those dependencies (activerecord etc.) are development ones, only declared as runtime in plugins that depend on `gem killbill`. but yeah, here we go https://github.com/killbill/killbill-plugin-framework-ruby/commit/adc36d77e8c39d82d308a07334d1f6542cfde4d3